sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/delete_key_operation.hpp

```diff
@@ -35,7 +36,7 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
      * constructor is private and requires internal components.*/
     friend class KeyClient;
```

Comment:
Can we remove this now?

Reply:
No, an SDK client still need access to the other constructor to create an operation.

Basically, there are 2 ways of creating an Operation, by calling a client API (like StartFoo()) , which makes a client need to access operation private ctor. And with static factory method from the OperationFoo class from a resume token